File: codes/java/factorial/Factorial.java
import java.util.Scanner;

class Factorial {
    public static void main(String[] args) {
        int n,i, fect=1;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        n = sc.nextInt();
        sc.close();
        for(i=1; i<=n; i++) {
            fect = fect * i;
        }
        System.out.println("Factorial is " + fect);
    }
}

File: codes/c++/factorial/factorial.cpp
#include<iostream>
using namespace std;

int main() {
    int n, i, fect=1;
    cout<<"Enter the number: ";
    cin>>n;
    for(i=1; i<=n; i++) {
        fect = fect * i;
    }
    cout<<"Factorial: "<<fect;
    return 0;
}